Modules/Options/TrackerTab/QuestieOptionsTracker.lua

```diff
@@ -514,22 +545,22 @@ function QuestieOptions.tabs.tracker:Initialize()
                 type = "select",
                 order = 3.2,
                 values = function() return {
-                        ["AUTO"] = l10n('Auto'),
-                        ["TOPLEFT"] = l10n('Top Left'),
-                        ["TOPRIGHT"] = l10n('Top Right'),
-                        ["BOTTOMLEFT"] = l10n('Bottom Left'),
-                        ["BOTTOMRIGHT"] = l10n('Bottom Right'),
+                        ["TOPLEFT"] = l10n('Down'),
+                        ["BOTTOMLEFT"] = l10n('Up'),
                     }
                 end,
                 style = 'dropdown',
-                name = function() return l10n('Tracker SetPoint'); end,
-                desc = function() return l10n("This determines the direction in which the tracker grows. The default setting will adjust based on where you drag the tracker automatically or you can override it and force it to populate however you like.\n\nFor example, if you select 'Bottom Left' then the tracker will grow from Left to Right and Quests will be populated from the Bottom of the Tracker to the Top. This setting is ideal when you want the tracker to be placed in the 'Bottom Left' region of your UI etc.\n\nNOTE: If you change this setting then 'Auto Move Header' will be disabled and will remain at the top of the Tracker even while minimized."); end,
+                name = function() return l10n('Tracked Quests Grow'); end,
+                desc = function() return l10n("This determines the direction in which the tracker grows when you add or remove quests. This will also move the Active Quests Header to either the top of the Questie Tracker (when using the 'Down' setting) or the bottom of the Questie Tracker (when using the 'Up' setting). \n\nNOTE: You can override the Active Quests Header movement behavior by disabling the 'Auto Move Header' option in Questie Tracker Options to force the Active Quests Header to remain at the top of the Questie Tracker."); end,
```

Comment:
The [translation](https://github.com/Questie/Questie/tree/master/Localization/Translations) needs to be fixed